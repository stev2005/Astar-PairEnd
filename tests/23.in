TACGAAGTCAGGCATTGGTTCCTTTTTTCACCCGTCGCATCGGGGCCCTAACGTACTTAATCCATCCGGCCAATGGCATAAACGTCTGATTAAAAACGGATTAT
15
GGAGCATCGCCTCTAAGACTACAGTAGGACCTTCCCGAGAGTTTTTTCTAAACCGACATGGACCGGGATCACATA
TTCGACGCGCGACCG
CTCTCTTAATCCTCA
CCACTTGCAACCCTCTGGGACGAGCG
ATTGACTCAGACGATTGGACCGTCACCGTTCTTAGTCCCCTCTCCGATACACTAGTCACATGAGTGG
TTGGACTCTTCTACCACTTTTC
TTTCGCTTAGCGGGCACCCACGCCACCTAGGTCGCATATATAGCGACTCGTGCCACGCAGTGGAATAATTAGTGACG
ATAAGCCGGTTGCGATGCGTCGGCAGTATGGACGATTCCCAATCGTCCATACCGG
AGGAAAACGACCCGGCTCGCCTGCTGTACCGCTCCTCC
ACGCGAATTCGCGCTAAGTCTATAAAGTGTAGCGTTTTTGCCTTGTTTCGAATTATA
GGGTATCTGATCTAGGTCCTAGCAGATTTTCGCTCCGGCCTAGGACATGGGGAAGTACGACTGGTTACCCGACCGCGT
ACTGCTAATCGTGGCCCACGGGCGTGCAAAAACTGGGGTGTCCGTTTATATCGAACGCCGCCGGCAATGTCCCTTAGGCC
ATAATCGAGACAGTCGTAATGCACCATTCAGCTGCGATGGTTGCGTTGATCGCCTG
GCTGTAGGGAGCT
CGTGCGGTGCAAGCTAGCGCACGGTAGCCC
