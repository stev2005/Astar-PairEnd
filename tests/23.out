36
5
5
12
30
9
36
22
19
28
35
39
24
5
12
